gui/accelmap.py

```diff
@@ -70,20 +71,32 @@ def __init__(self):
         self.ui_manager = None
         self.connect("show", self._show_cb)
 
+        self.set_row_spacing(5)
+
         store = Gtk.ListStore(*self._COLUMN_TYPES)
         self._store = store
         self._action_labels = {}
         self._accel_labels = {}
 
+        self._filter_entry = Gtk.Entry()
+        self._filter_entry.set_placeholder_text(_('Filter'))
```

Comment:
Import and use the `C_` function instead of this one, and provide "keymap-editor" (or similar) as the first argument. 

The first argument is the "context", distinguishing multiple appearances of the same word, as that word might have different translations in different situations, for some languages.